charts/docker-hub-rate-limit-exporter-chart/values.yaml

```diff
@@ -33,29 +60,30 @@ serviceAccount:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
-
-# Labels to add to each pod.
-podLabels: {}
-    # myPodLabel: labelName
+    imagePullSecrets: []
 
 podAnnotations: {}
 
-podSecurityContext:
-    {}
-    # fsGroup: 2000
-
 securityContext:
-    {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  fsGroup: 65534
+  runAsGroup: 65534
+  runAsNonRoot: true
+  runAsUser: 65534
+
+containerSecurityContext: {}
+  # capabilities:
+  #   add:
+  #   - SYS_TIME
 
 service:
-    type: ClusterIP
-    port: 80
+  type: ClusterIP
+  port: 9100
```

Comment:
Is there a reason you changed to port 9100? I'm fine with this change, I'm just curious.

Reply:
Its just because i saw prometheus node exporter use this port, but i dont have any proper opinion on this.
I will keep it to 80 since `TCP port 80, most often used by Hypertext Transfer Protocol used to send and receive unencrypted web pages`